numpy/core/src/umath/ufunc_object.c

```diff
@@ -5572,6 +5603,8 @@ ufunc_get_identity(PyUFuncObject *ufunc)
         return PyInt_FromLong(1);
     case PyUFunc_Zero:
         return PyInt_FromLong(0);
+    case PyUFunc_MinusOne:
```

Comment:
Why are these values not the same as the ones returned by `assign_reduce_identity_*()`? Specifically, `assign_reduce_identity_minusone()` uses `PyLong_FromLong()` instead of `PyInt_FromLong()`. It looks like one is a macro for the other in Py3, but is that the case in Py2 as well?